app/src/test/java/org/wikipedia/test/ParcelableTest.kt

```diff
@@ -41,6 +41,8 @@ class ParcelableTest {
     fun testPagePropertiesFromSummary() {
         val json = TestFileUtil.readRawFile("rb_page_summary_geo.json")
         val summary = JsonUtil.decodeFromString<PageSummary>(json)!!
+        // FIXME: somehow the Location object is different even though the values are the same. Set null to bypass the test temporary
```

Comment:
This test fails because the `PageProperties` class contains a `Location` field, which is not a primitive type. If you parcel and then unparcel a `PageProperties` object, the actual instance of the Location field will be different in the unparcelled object than the parcelled object.

The issue is really that the test isn't smart enough to perform a "deep-equal" between the two objects, and instead it's just comparing the literal instances.

I'm actually not sure I see the point of these tests anyway. We're testing whether `Parcelable` objects can be parcelled and unparcelled... Are we testing whether the Android SDK works? Whether the Java language works? What exactly could fail here?
Anyway, I think the fix in this PR is good to merge.